packages/sdk/src/expression.test.ts

```diff
@@ -346,6 +411,42 @@ describe("transpile expression", () => {
     }
     expect(errorString).toEqual(`Unexpected token (1:0) in ""`);
   });
+
+  test("transpile string methods with optional chaining", () => {
+    expect(
+      transpileExpression({
+        expression: "title.toLowerCase()",
+        executable: true,
+      })
+    ).toEqual("title?.toLowerCase()");
```

Comment:
It should add optional chaining to function calls as well so calling method on some object would not give runtime errors. `title?.toLowerCase?.()`. We do the same in collection `array?.map?.(item`